xdsl/dialects/linalg.py

```diff
@@ -239,6 +241,10 @@ def print(self, printer: Printer):
         printer.print_string(" ")
         printer.print_region(self.body)
 
+        if self.res:
+            printer.print_string(" -> ")
+            printer.print_list(self.res, lambda res: printer.print_attribute(res.type))
+
```

Comment:
This and the parsing code below adds the result type to the custom format, can be done in the same PR as #2115 or as a follow-up

Reply:
ill add this in the same pr as #2115